sympy/utilities/tests/test_lambdify.py

```diff
@@ -1689,3 +1690,18 @@ def test_23536_lambdify_cse_dummy():
     eval_expr = lambdify(((f, g), z), expr, cse=True)
     ans = eval_expr((1.0, 2.0), 3.0)  # shouldn't raise NameError
     assert ans == 300.0  # not a list and value is 300
+
+def test_lambdify_arb():
+    if not flint:
+        skip("flint not installed.")
+
+    f1 = lambdify([x, y], x**3-y**3, modules='arb')
+    a1 = f1(10**15, 10**15 - 1)
+    a1_lo = float(flint.arb.abs_lower(a1))
+    a1_hi = float(flint.arb.abs_upper(a1))
+    assert math.isclose(a1_lo, 2.8610732685849445e+30)
+    assert math.isclose(a1_hi, 3.160267082499145e+30)
```

Comment:
I still haven't found a good way of testing an Arb: https://github.com/fredrik-johansson/python-flint/pull/39#issuecomment-1357580852